src/Cms/App.php

```diff
@@ -1414,7 +1437,11 @@ protected function setSite($site = null)
      */
     public function server()
     {
-        return $this->server = $this->server ?? new Server();
+        if ($this->server !== null) {
+            return $this->server;
+        }
+
+        return $this->server = new Server();
```

Comment:
These whole changes could be reverted now that we don't do anything else in this method.

Reply:
Oh you are right. I guess it should probably even be: 

```
return $this->server ??= new Server();
```